src/modules/task/rest-api/task-access-middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express';
+import TaskService from '../task-service';
+import { Task, UnAuthorizedTaskAccessError } from '../types';
+
+export default class TaskAccessMiddleware {
```

Comment:
Why is this required? We'll validate the access token and only fetch tasks for that account.

Reply:
@jjalan  sir, told me that we need a middleware to ensue the taskId we are passing should be of accountId we are passing in request params.